components/02-molecules/accordion/accordion.twig

```diff
@@ -0,0 +1,53 @@
+{% set accordion__base_class = 'accordion' %}
+
+{% set expand_all__content %}
+  Expand All
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'angle-down',
+    icon__blockname: accordion__base_class,
+  } %}
+{% endset %}
+
+{% set collapse_all__content %}
+  Collapse All
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'angle-down',
+    icon__blockname: accordion__base_class,
+  } %}
+{% endset %}
+
+<div {{ bem(accordion__base_class) }} data-component-width="content">
+  <div {{ bem('inner', [], accordion__base_class) }}>
+    {% include "@atoms/typography/headings/heading.twig" with {
+      heading__level: '3',
```

Comment:
- [ ]  `accordion__heading`should be a `<h2>`, since the `accordion-item__headings` are `<h3>`s. This will ensure proper nesting. 

<img width="496" alt="Screen Shot 2022-03-25 at 10 17 24 AM" src="https://user-images.githubusercontent.com/12991593/160138116-3bab69b8-2646-405f-84fe-27df7fff50ba.png">

Reply:
Also, quick question, how will the `accordion__heading` be filled in?

Reply:
I've got the conditional heading levels done. (also conditionally showing the top-level heading markup only if a heading is provided)

@kara-franco The heading will be filled in when someone places an Accordion on the page.
- They'll be asked if they want to give a name to their accordion grouping
- Then they'll be able to add accordion items

Reply:
Very nice, thank you!